common/meta/types/src/user_grant.rs

```diff
@@ -28,10 +28,20 @@ pub enum GrantObject {
 }
 
 impl GrantObject {
-    /// Some global privileges can not be granted to a database or table, for example, a KILL
-    /// statement is meaningless for a table.
-    pub fn allow_privilege(&self, privilege: UserPrivilegeType) -> bool {
-        self.available_privileges().has_privilege(privilege)
+    /// Comparing the grant objects, the Database object contains all the Table objects inside it.
+    /// Global object contains all the Database objects.
+    pub fn contains(&self, object: &GrantObject) -> bool {
+        use GrantObject::*;
+        match (self, object) {
+            (Global, _) => true,
+            (Database(_), Global) => false,
```

Comment:
`GrantObject::Database` instead of `use GrantObject::*;` looks better

Reply:
thank you, fixed